phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
Yup, @chancez convinced me. If you remove the superfluous line above (and my current test deployment looks okay) then I'll merge it.